
syntax = "proto3";
package c2s;

message UserRegister
{
    message Request
    {
        string Account = 1;  //账户
        string PassWord = 2; //密码
        int64 PhoneNum = 3; //手机号
        string Platform = 4;    //平台
        string DeviceMac = 5;    //设备物理地址
    }
    message Response
    {
        int64 UserId = 1;   //用户id
    }
}

message UserVerify
{
    message Request
    {
        string Account = 1; //账户
        string PassWord = 2;   //密码
        string Token = 3;   // 登录token
    }
    message Response
    {
        int64 UserId = 1;   //玩家id
        string Token = 2;   //新的token
        string GateAddress = 3; //网关地址
    }
}